fastn/src/main.rs

```diff
@@ -80,7 +83,7 @@ async fn fastn_core_commands(matches: &clap::ArgMatches) -> fastn_core::Result<(
         let inline_css = serve.values_of_("css");
         let offline = serve.get_flag("offline");
 
-        fastn_update::update(&config, offline).await?;
+        fastn_update::update(&ds, offline).await?;
```

Comment:
Since config is not needed by fastn_update, we can re-order this function to not construct config before calling this.